src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -0,0 +1,87 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import frc.robot.subsystems.Drivetrain;
+import static frc.robot.Constants.LimelightConstants.*; 
+
+public class Limelight extends SubsystemBase {
+  /**
+   * Creates a new Limelight.
+   */
+
+   private Drivetrain m_drivetrain;  // Add a local variable for the Drivetrain
+
+  //Create variables for the different values given from the limelight
+  public double tx;
+  public double ty;
+  public double ta;
+  public double tv;
+
+  // Create a network table for the limelight
+  private final NetworkTable m_limelightTable;
+
+  // Returns a value of the offset on the x-axis of the camera to the target
+  public double getTX() {
+    return tx;
+  }
+
+  // Returns a value of 0 or 1 depending on whether or not a target is detected
+  public double getTV() {
+    return tv;
+  }
+
+  // Returns true if a target is detected.
+  public boolean isTargetDetected() {
+    return (tv > 0.0);
+  }
```

Comment:
These effectively do the same thing.  Is there a reason we'll need the numeric value of tv, or is the boolean enough?  If so, let's just keep the boolean, since that feels more practical.